.travis.yml

```diff
@@ -3,6 +3,8 @@ sudo: false
 python:
   - 2.7
 install:
+  - virtualenv .
```

Comment:
I thought travis gives you a virtualenv by default? I mean, I remember deactivating the virtualenv in one of my `.travis.yml` files in order to access some system packages :-)

Reply:
If you try this branch on travis without the last commit it fails with a setuptools version conflict, see: https://travis-ci.org/reinout/z3c.dependencychecker/builds/279908297

Reply:
Looking at it in more detail it seems that:

- travis has setuptools **36.3.0** installed
- it installs (via requirements.txt) version **33.1.1**
- pip detects 36.3.0 installed and uninstalls it (or so it says...)
- on the final overview of the pip install -r requirements-dev.txt it says that setuptools **36.0.1** is installed
- then buildout starts and has a conflict between 33.1.1 and 36.0.1

No idea why this 36.0.1 comes from...